src/routes/spaces/spaces.service.ts

```diff
@@ -136,7 +136,7 @@ export class SpacesService {
     const space = await this.spacesRepository.findOneOrFail({
       where: {
         id,
-        userOrganizations: { user: { id: userId } },
+        userOrganizations: { user: { id: userId, status: 'ACTIVE' } },
```

Comment:
Nit: what do you think about renaming this function to be explicit, e.g. `getActiveSpace`?

Reply:
Good point as well, I've modified that function name, and also `get` to `getActiveOrInvitedSpaces` to be more explicit in both cases.